integration/testdata/fixtures/sbom/license-cyclonedx.json

```diff
@@ -0,0 +1,1311 @@
+{
+  "$schema": "http://cyclonedx.org/schema/bom-1.5.schema.json",
+  "bomFormat": "CycloneDX",
+  "specVersion": "1.5",
+  "serialNumber": "urn:uuid:b88b712b-e794-4216-a414-771ef5083e82",
+  "version": 1,
+  "metadata": {
+    "timestamp": "2024-02-19T19:41:18+00:00",
+    "tools": {
+      "components": [
+        {
+          "type": "application",
+          "group": "aquasecurity",
+          "name": "trivy",
+          "version": "dev"
+        }
+      ]
+    },
+    "component": {
+      "bom-ref": "31804a3a-5d7d-4be1-8536-8e28bdd9eccd",
+      "type": "application",
+      "name": "C:/Users/bedla.czech/IdeaProjects/sbom-demo",
+      "properties": [
+        {
+          "name": "aquasecurity:trivy:SchemaVersion",
+          "value": "2"
+        }
+      ]
+    }
+  },
+  "components": [
```

Comment:
I think there is no need for so many dependencies for this test.
3-4 will be enough